spec/models/badge_spec.rb

```diff
@@ -31,35 +31,6 @@
     end
   end
 
-  context "when callbacks are triggered after save" do
```

Comment:
I have actually no idea what this test does and why it is failing. Maybe because of cache?

Reply:
Yup, this test was meant to ensure the the callback triggers (in this case `cache_bust`) were being called. But since they were removed it's no longer needed 👍🏼